spec/rubocop/cop/lint/duplicate_set_enumerables_spec.rb

```diff
@@ -0,0 +1,75 @@
+# frozen_string_literal: true
+
+RSpec.describe RuboCop::Cop::Lint::DuplicateSetEnumerables, :config do
+  context 'when there is duplicated symbols in set' do
+    it 'registers an offense' do
+      expect_offense(<<~RUBY)
+        Set.new[:otherkey, :key, :key]
+                                 ^^^^ Duplicate enumerables found in Set.
+                           ^^^^ Duplicate enumerables found in Set.
```

Comment:
consider using `consecutive_duplicates` instead of `duplicates` like [here](https://github.com/rubocop/rubocop/blob/b03a1edd08037f61641ca76d81e7804ca3f404d5/lib/rubocop/cop/lint/duplicate_hash_key.rb#L28)

this would produce a much [cleaner error message](https://github.com/rubocop/rubocop/blob/b03a1edd08037f61641ca76d81e7804ca3f404d5/spec/rubocop/cop/lint/duplicate_hash_key_spec.rb#L7-L8) where only the first duplicate is highlighted.

In general, I think this cop is extremely similar to `Lint/DuplicateHashKey`, so we should probably follow its conventions to keep things consistent 🙂